content/docs/introduction/faq.mdx

```diff
@@ -188,7 +199,7 @@ Everything in the `tina` directory (except the `__generated__` folder) needs to
 
 ### How do I resolve `API URL is misformatted` errors?
```

Comment:
Hi @bradystroud

- [ ] Change from

> How do I resolve `API URL is misformatted` errors?

to

> How to resolve `API URL is misformatted` errors?

Reply:
Disagree - the suggestion is not a question